zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/AbstractZipkinFactory.java

```diff
@@ -145,27 +148,44 @@ private static int toInt(final String ip) {
     }
 
     /**
-     * Build a new {@link Brave} instance for interfacing with Zipkin
+     * Build a new {@link Tracing} instance for interfacing with Zipkin
      *
      * @param environment
      *            Environment
      * @param reporter
      *            reporter
-     * @return Brave instance
+     * @return Tracing instance
      */
-    protected Optional<Brave> buildBrave(@Nonnull final Environment environment,
+    protected Optional<Tracing> buildTracing(
+            @Nonnull final Environment environment,
             @Nonnull final Reporter<Span> reporter) {
 
         LOGGER.info("Registering Zipkin service ({}) at <{}:{}>", serviceName,
                 serviceHost, servicePort);
 
-        final Brave brave = new Brave.Builder(toInt(serviceHost), servicePort,
-                serviceName).reporter(reporter).traceSampler(getSampler())
-                        .traceId128Bit(traceId128Bit).build();
+        final Endpoint endpoint = Endpoint.builder().ipv4(toInt(serviceHost))
+                .port(servicePort).serviceName(serviceName).build();
 
-        // Register the tracing feature for client and server requests
-        environment.jersey().register(BraveTracingFeature.create(brave));
+        final Tracing tracing = Tracing.newBuilder()
+                .currentTraceContext(MDCCurrentTraceContext.create())
+                .localServiceName(serviceName).localEndpoint(endpoint)
+                .reporter(reporter).sampler(getSampler())
+                .traceId128Bit(traceId128Bit).build();
 
-        return Optional.of(brave);
+        // Register the tracing feature for client and server requests
+        environment.jersey().register(TracingFeature.create(tracing));
```

Comment:
you might want to accept HttpTracing, as this allows you to customize span name and tagging policy https://github.com/openzipkin/brave/tree/master/instrumentation/http#span-data-policy